<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDataSetting</class>
 <widget class="QDialog" name="DialogDataSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Setting</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>450</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_EMG">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>171</y>
     <width>411</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>EMG</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <widget class="QGroupBox" name="groupBox_DataRate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>371</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Data Rate</string>
    </property>
    <widget class="QRadioButton" name="radioButton_650Hz">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>650Hz</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_500Hz">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>500Hz</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_250Hz">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>250Hz</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_100Hz">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>100Hz</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_200Hz">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>200Hz</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_400Hz">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>400Hz</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_1000Hz">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1000Hz</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_DataBits">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Data Bits</string>
    </property>
    <widget class="QRadioButton" name="radioButton_12bit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>12bits</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_8bit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>8bits</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Gyro">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>411</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Gyro</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <widget class="QGroupBox" name="groupBox_GyroDataRate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Data Rate</string>
    </property>
    <widget class="QRadioButton" name="radioButton_Gyro100HZ">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>100HZ</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Gyro50HZ">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>50HZ</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Accel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>411</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Accel</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <widget class="QGroupBox" name="groupBox_AccelDataRate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Data Rate</string>
    </property>
    <widget class="QRadioButton" name="radioButton_Accel100HZ">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>100HZ</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Accel50HZ">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>50HZ</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Magnetic">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>411</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Magnetic</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <widget class="QGroupBox" name="groupBox_MagneticDataRate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Data Rate</string>
    </property>
    <widget class="QRadioButton" name="radioButton_Magnetic10HZ">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>10HZ</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Quaternion">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>411</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Quaternion</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QGroupBox" name="groupBox_QuaternionDataRate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Data Rate</string>
    </property>
    <widget class="QRadioButton" name="radioButton_Quaternion100HZ">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>100HZ</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Quaternion50HZ">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>50HZ</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Note: Data will be saved to combined file when recording if data type checked besides EMG.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>groupBox_Quaternion</zorder>
  <zorder>groupBox_Magnetic</zorder>
  <zorder>groupBox_EMG</zorder>
  <zorder>buttonBox</zorder>
  <zorder>groupBox_Gyro</zorder>
  <zorder>groupBox_Accel</zorder>
  <zorder>label</zorder>
 </widget>
 <tabstops>
  <tabstop>groupBox_Accel</tabstop>
  <tabstop>radioButton_Accel50HZ</tabstop>
  <tabstop>radioButton_Accel100HZ</tabstop>
  <tabstop>groupBox_Gyro</tabstop>
  <tabstop>radioButton_Gyro50HZ</tabstop>
  <tabstop>radioButton_Gyro100HZ</tabstop>
  <tabstop>groupBox_Magnetic</tabstop>
  <tabstop>radioButton_Magnetic10HZ</tabstop>
  <tabstop>groupBox_Quaternion</tabstop>
  <tabstop>radioButton_Quaternion50HZ</tabstop>
  <tabstop>radioButton_Quaternion100HZ</tabstop>
  <tabstop>groupBox_EMG</tabstop>
  <tabstop>radioButton_8bit</tabstop>
  <tabstop>radioButton_12bit</tabstop>
  <tabstop>radioButton_100Hz</tabstop>
  <tabstop>radioButton_200Hz</tabstop>
  <tabstop>radioButton_250Hz</tabstop>
  <tabstop>radioButton_400Hz</tabstop>
  <tabstop>radioButton_500Hz</tabstop>
  <tabstop>radioButton_650Hz</tabstop>
  <tabstop>radioButton_1000Hz</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogDataSetting</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogDataSetting</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
